### ISSUE TITLE
Problem with LinearMaps

### ISSUE THREAD

**Issue by davkovacs** [ACEsuit/ACE.jl#115]
I tried to update ACE.jl to the latest commit, but couldn't because of the following error:
```
(@v1.6) pkg> update ACE
    Updating registry at `~/.julia/registries/MolSim`
    Updating git-repo `https://github.com/JuliaMolSim/MolSim.git`
ERROR: expected package `LinearMaps [7a12625a]` to be registered
```
@cortner do you happen to know what is going on here?

**Comment by cortner** [comment]
you don't have the General registry installed

**Comment by davkovacs** [comment]
thanks. Seems like the newest ACE does not work with ACEatoms though because of dependency conflicts. 
Any particular reason for this? Or is it safe to change version bounds in ACEatoms? 
```
ERROR: Unsatisfiable requirements detected for package ACEbase [14bae519]:
 ACEbase [14bae519] log:
 ├─possible versions are: 0.2.2 or uninstalled
 ├─restricted to versions 0.2 by JuLIP [945c410c], leaving only versions 0.2.2
 │ └─JuLIP [945c410c] log:
 │   ├─possible versions are: 0.13.3 or uninstalled
 │   └─JuLIP [945c410c] is fixed to version 0.13.3
 ├─ACEbase [14bae519] is fixed to version 0.2.2
 └─restricted to versions 0.2.4-0.2 by ACE [3e8ccfd2] — no versions left
   └─ACE [3e8ccfd2] log:
     ├─possible versions are: 0.12.25 or uninstalled
     ├─restricted to versions 0.12.30 by ACEatoms [1e34e032] — no versions left
     │ └─ACEatoms [1e34e032] log:
     │   ├─possible versions are: 0.0.8 or uninstalled
     │   └─ACEatoms [1e34e032] is fixed to version 0.0.8
     └─ACE [3e8ccfd2] is fixed to version 0.12.25
```

**Comment by cortner** [comment]
I don't know :) - try it out. And if the tests fail, then we need to update ACEatoms.jl.

**Comment by cortner** [comment]
> ACEbase [14bae519] is fixed to version 0.2.2

This looks odd - does it mean you pinned ACEbase?
